### ISSUE TITLE
Unable to generate docs for TargetFramework NS1.3 projects

### ISSUE THREAD

**Issue by Still34** [dotnet/docfx#2841]
**DocFX Version Used**: 
v2.36.1
**Template used**: (`default` or `statictoc` or contain custom template)
`default`/`custom`
**Steps to Reproduce**:

1. Use a project that targets multiple frameworks such as [Discord.NET](https://github.com/RogueException/Discord.Net) (Does this have to do with condition based `TargetFrameworks`?)
2. Attempt to build the project

The projects have failed to build on my own devices as well as on AppVeyor. 

**This does not happen in prior builds to VS15.7.**

**Expected Behavior**:
Succeed with the build
**Actual Behavior**:
Fail to build properly with warnings such as,
```[18-06-05 08:22:56.295]Warning:[MetadataCommand.ExtractMetadata](C:/Users/34146/source/repos/discord.net.fork/src/Discord.Net.Analyzers/Discord.Net.Analyzers.csproj)Workspace failed with: [Failure] Msbuild failed when processing the file 'C:\Users\34146\source\repos\discord.net.fork\src\Discord.Net.Core\Discord.Net.Core.csproj' with message: c:\program files\dotnet\sdk\2.1.201\Sdks\Microsoft.NET.Sdk\build\Microsoft.PackageDependencyResolution.targets: (167, 5): Assets file 'C:\Users\34146\source\repos\Discord.Net.Fork\src\Discord.Net.Core\obj\project.assets.json' doesn't have a target for '.NETStandard,Version=v1.3'. Ensure that restore has run and that you have included 'netstandard1.3' in the TargetFrameworks for your project.```

**Comment by Still34** [comment]
To clarify, `docfx.json` has been set up with 
```json

  "metadata": [{
    "src": [{
      "src": "../src",
      "files": [
        "**.csproj"
      ]
    }],
    "dest": "api",
    "filter": "filterConfig.yml",
    "properties": {
      "TargetFramework": "netstandard1.3"
    }
  }],
```

**Comment by Still34** [comment]
Updated .NET Core SDK to 2.1.300, the exceptions thrown are now NREs.

```
[18-06-13 06:09:38.699]Warning:[MetadataCommand.ExtractMetadata](C:/Users/34146/source/repos/discord.net.fork/src/Discord.Net.Analyzers/Discord.Net.Analyzers.csproj)Workspace failed with: [Failure] Msbuild failed when processing the file 'C:\Users\34146\source\repos\discord.net.fork\src\Discord.Net.Analyzers\Discord.Net.Analyzers.csproj' with message: c:\program files\dotnet\sdk\2.1.300\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets: (198, 5): Error loading lock file 'C:\Users\34146\source\repos\discord.net.fork\src\Discord.Net.Analyzers\obj\project.assets.json' : Object reference not set to an instance of an object.
```

**Comment by Still34** [comment]
Fixed in v2.36.2.
